BitwardenShared/Core/Platform/Utilities/SystemDevice.swift

```diff
@@ -14,6 +14,9 @@ protocol SystemDevice {
 
     /// The version of the operating system on the device, e.g. "17.0".
     var systemVersion: String { get }
+
+    /// Whether the device is an iPhone.
+    var isIPhone: Bool { get }
```

Comment:
🎨 Tests are failing because `MockSystemDevice` doesn't have the conformance to this new property, could you add this getter property to it?